### DISCUSSION TITLE
Move and resize lag

### DISCUSSION THREAD

**Discussion by garethw14** [rabbibotton/clog#264] [General]
Hello Cloggers,

I have been putting together a nice website app with CLOG and the builder.  

It all works silky-smooth locally, but I intend for this to be used remotely, and when I deployed it onto a cloud service, I was surprised to find that moving and resizing CLOG Windows incurred a fair amount of lag.   While it's clear that anything where you have to talk back to the server is going to incur network overhead, it's not obvious to me why you'd have to do so at the beginning of these operations in my mental model of how this all works.  

It seems like when the move or resize is initiated, we block while talking over the websocket.  How come?

Is there a way to make these operations less choppy?

Am I missing something?  

PS. The CLOG Plunger video series were such a great kick-start to getting productive.  Highly appreciated and recommended!

**Comment by rabbibotton** [comment]
If using clog-gui when creating the windows that can be dragged use - :client-movement t
That should solve this issue. Let me know

**Reply by garethw14** [reply]
Perfect!  I now recall seeing that at some point.  Silky-smoothness restored.

**Reply by rabbibotton** [reply]
The main advantage of setting that f is for local apps as you have access to more events.

**Comment by rabbibotton** [comment]
"PS. The CLOG Plunger video series were such a great kick-start to getting productive. Highly appreciated and recommended!"

Thanks :)

I still plan at some point adding a few more videos to it. Last month and a half had happy things to take care of.
